src/Materials/effectRenderer.ts

```diff
@@ -8,6 +8,7 @@ import { Constants } from '../Engines/constants';
 import { Observable } from '../Misc/observable';
 import { Effect } from './effect';
 import { DataBuffer } from '../Meshes/dataBuffer';
+import { DrawWrapper } from "./drawWrapper";

```

Comment:
Not sure about why we need the drawwrapper here ?

Reply:
That's because `EffectRenderer` is calling `engine.enableEffect` and this call should always be made with a `DrawWrapper` as the parameter for WebGPU to work as expected.